lib/mayaUsd/fileio/enhancedSparseValueWriter.h

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef PXRUSDMAYA_ENHANCED_SPARSE_VALUE_WRITER_H
+#define PXRUSDMAYA_ENHANCED_SPARSE_VALUE_WRITER_H
+
+#include <mayaUsd/base/api.h>
+
+#include <pxr/base/vt/value.h>
+#include <pxr/pxr.h>
+#include <pxr/usd/usd/attribute.h>
+#include <pxr/usd/usd/timeCode.h>
+#include <pxr/usd/usdUtils/sparseValueWriter.h>
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+/// Enhanced spare value writer.
```

Comment:
Enhanced does not feel right. We are removing a feature of the sparse value writer. Can you consider "FlexibleValueWriter", or the more traditional "MayaUsdSparseValueWriter"?